### ISSUE TITLE
guide: Running Experiments improvements

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/dvc.org#2768]
> Follow up to https://github.com/iterative/dvc.org/pull/2690

- [ ] Extract unnecessary details from https://dvc.org/doc/command-reference/exp (mainly from `run`) since a lot of it is explained here now.
- [ ] The pipeline section seems too deep. I think the guide could link to other parts of the docs more liberally -- for now https://dvc.org/doc/start/data-pipelines or https://dvc.org/doc/command-reference/dag#directed-acyclic-graph?
- [ ] Add about metrics.
- [ ] Explain that experiments can also track changes to data, code, etc.
- [ ] Delete note on "each experiment runs in an independent temporary directory" ?
- [ ] Review conceptual level of each section e.g. H2s for main subtopics, H3s for methods, hidden blocks for implementation `<details>`
- [ ] **Checkpoint experiments** is basically pulled from https://dvc.org/doc/command-reference/exp/run#checkpoints. We need to determine what to leave on the cmd ref. (rel 32548)

**Comment by iesahin** [comment]
I believe the command reference should contain examples with one or two sentence explanations and the rest should be moved to UG. Command examples can continue to live in the ref. WDYT @jorgeorpinel

**Comment by jorgeorpinel** [comment]
Yes, we need some cmd ref examples focused on how to use the CLI (not so much on explaining concepts). There can be examples in both places.
